docs/examples/NavbarForm.js

```diff
@@ -0,0 +1,21 @@
+const navbarInstance = (
+  <Navbar>
+    <Navbar.Header>
+      <Navbar.Brand>
+        <a href="#">Brand</a>
+      </Navbar.Brand>
+      <Navbar.Toggle />
+    </Navbar.Header>
+    <Navbar.Collapse>
+      <Navbar.Form pullLeft>
+        <div className="form-group">
+          <Input type="text" placeholder="Search"/>
```

Comment:
Not without a label prop I think...I can double check but the markup looked good when I looked at it originally